src/ui/src/utils/launchDarklyUtils.ts

```diff
@@ -0,0 +1,96 @@
+export type LaunchDarklyEnvironment = "Development" | "Production" | "Test";
+
+const DEFAULT_ENVIRONMENT: LaunchDarklyEnvironment = "Development";
+
+export function getLaunchDarklyClientId(): string | undefined {
+	return typeof window !== "undefined"
+		? window.__WRITER_APP_CONFIG__?.LAUNCH_DARKLY
+		: undefined;
```

Comment:
`window.__WRITER_APP_CONFIG__` is a base64 string on prod (https://app.writer.com/env/config.js).


```suggestion
  if (typeof window !== "undefined") return;
  if (typeof window.__WRITER_APP_CONFIG__ === 'string') {
    try {
      return JSON.parse(atob(window.__WRITER_APP_CONFIG__)).LAUNCH_DARKLY
    } catch {
      return undefined
    }
  } elseif (typeof window.__WRITER_APP_CONFIG__ === 'object' && window.__WRITER_APP_CONFIG__) {
    return window.__WRITER_APP_CONFIG__.LAUNCH_DARKLY
  }
  return undefined;
```